addons/hellfire/CfgWeapons.hpp

```diff
@@ -29,4 +30,26 @@ class CfgWeapons {
         magazines[] = {"6Rnd_ACE_Hellfire_AGM114N", "PylonMissile_1Rnd_ACE_Hellfire_AGM114N", "PylonRack_1Rnd_ACE_Hellfire_AGM114N", "PylonRack_3Rnd_ACE_Hellfire_AGM114N", "PylonRack_4Rnd_ACE_Hellfire_AGM114N"};
         magazineWell[] += {QGVAR(N)};
     };
+    class GVAR(launcher_L): GVAR(launcher) {
+        displayName = "AGM-114L Hellfire ""Longbow""";
+        magazines[] = {"6Rnd_ACE_Hellfire_AGM114L", "PylonMissile_1Rnd_ACE_Hellfire_AGM114L", "PylonRack_1Rnd_ACE_Hellfire_AGM114L", "PylonRack_3Rnd_ACE_Hellfire_AGM114L", "PylonRack_4Rnd_ACE_Hellfire_AGM114L"};
+        magazineWell[] += {QGVAR(L)};
+        EGVAR(laser,showHud) = 1;
```

Comment:
```suggestion
        EGVAR(laser,showHud) = 1; // Just to show the attack profile
```
I think this is why; maybe something we could look at in the future

Reply:
Yeah that's why. I was thinking of ripping out the missileguidance stuff from laser but I decided against it for this PR